pal/samples/esp8266/iothub_ll_telemetry_sample/iothub_ll_telemetry_sample.ino

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <AzureIoTHub.h>
+
+// CAVEAT: This sample is to demonstrate azure IoT client concepts only and is not a guide design principles or style
+// Checking of return codes and error values shall be omitted for brevity.  Please practice sound engineering practices
+// when writing production code.
+
+// Note: PLEASE see https://github.com/Azure/azure-iot-arduino#simple-sample-instructions for detailed sample setup instructions.
+// Note2: To use this sample with the esp32, you MUST build the AzureIoTSocket_WiFi library by using the make_sdk.py,
+//        found in https://github.com/Azure/azure-iot-pal-arduino/tree/master/build_all. 
+//        Command line example: python3 make_sdk.py -o <your-lib-folder>
+
+#include <stdio.h>
+#include <stdlib.h>
+
+// You must set the device id, device key, IoT Hub name and IotHub suffix in
+// iot_configs.h
+#include "iot_configs.h"
+#include "sample_init.h"
+#ifdef is_esp_board
+  #include "Esp.h"
+#endif
+
+
+static char ssid[] = IOT_CONFIG_WIFI_SSID;
+static char pass[] = IOT_CONFIG_WIFI_PASSWORD;
+
+#ifdef SAMPLE_MQTT
+    #include "AzureIoTProtocol_MQTT.h"
+    #include "iothubtransportmqtt.h"
+#endif // SAMPLE_MQTT
+#ifdef SAMPLE_HTTP
+    #include "AzureIoTProtocol_HTTP.h"
+    #include "iothubtransporthttp.h"
+#endif // SAMPLE_HTTP
```

Comment:
Can we more header files at top ?
Structure the code something like this:

<headerfiles>

global variables


functions

Reply:
Yes